### ISSUE TITLE
parket recompute the views too many times

### ISSUE THREAD

**Issue by tinchoz49** [ForsakenHarmony/parket#8]
Hi @ForsakenHarmony! how are you?

I've been testing the library and I saw that for only one mutation a view is recomputed four times. This can be a performance issue if you are working with expensive computations like filtering arrays.

https://repl.it/@MartinAcosta/views-parket
```javascript
import model from 'parket';

const Person = model('Person', { // name is used internally for serialization
  initial: () => ({
    firstname: null,
    lastname: null
  }),
  actions: state => ({
    setFirstName (first) {
      state.firstname = first; // no set state, no returns to merge, it's reactive™
    },
    setLastName (last) {
      state.lastname = last;
    }
  }),
  views: state => ({
    fullname: () => {
      console.log('fullname');
      return `${state.firstname} ${state.lastname}`;
    }
  }),
});

const instance = Person({ firstname: 'Tom' });
instance.setLastName('Clancy');
```

**Comment by ForsakenHarmony** [comment]
hey, I'm doing alright

I'll look into it, that's definitely too many updates
Sadly can't really do magic and check which props are used in the view

**Comment by gabemeola** [comment]
Would it be possible to pass state to each view, then diff and memoize?

**Comment by ForsakenHarmony** [comment]
I've released 0.2.1 which should only update the views twice in your example

**Comment by tinchoz49** [comment]
it works! :) thanks @ForsakenHarmony! closing the issue
